### ISSUE TITLE
Add CI test for 32-bit OS

### ISSUE THREAD

**Issue by enobufs** [pion/webrtc#750]
This happens after pion/webrtc@2.0.23...
- v2.0.23 - OK
- v2.0.24 - fail (TestDataChannelBufferedAmount/set_before_datachannel_becomes_open fail)
- v2.0.25 - fail (TestDataChannelBufferedAmount/set_before_datachannel_becomes_open fail)

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x804a7cc]

goroutine 48 [running]:
runtime/internal/atomic.Xadd64(0x90526ec, 0x4, 0x0, 0xffffffff, 0x0)
	/root/go/current/src/runtime/internal/atomic/asm_386.s:105 +0xc
github.com/pion/datachannel.(*DataChannel).WriteDataChannel(0x90526c0, 0x903dd90, 0x4, 0x4, 0x8070b00, 0x91204d0, 0x91d60c0, 0xc0000000)
	/root/go_ws/pkg/mod/github.com/pion/datachannel@v1.4.4/datachannel.go:294 +0x7e
github.com/pion/webrtc/v2.(*DataChannel).Send(0x9118360, 0x9130d90, 0x4, 0x4, 0x9120518, 0x9120a88)
	/root/go_ws/pkg/mod/github.com/pion/webrtc/v2@v2.0.24/datachannel.go:320 +0x72
```

I believe, the atomic operation (for datachannel stats we recently added) has some alignment issue. (not 100% sure yet)

This is causing crashes on win32 and android - also with 32bit.

**Comment by enobufs** [comment]
You can use 32-bit ubuntu docker image to repro it:

```
docker run -it i386/ubuntu /bin/bash
```

We should add ci test using 32-bit machine/os...

**Comment by enobufs** [comment]
I wonder if this tool can help... https://godoc.org/golang.org/x/tools/go/analysis/passes/atomicalign

I dunno how to use it yet...

**Comment by enobufs** [comment]
FYI. Using the docker image at `i386/ubuntu`, I was able to repro it (the test "TestDataChannelBufferedAmount/set_before_datachannel_becomes_open" failed).  After the fix (v2.0.16), all the pion/webrtc tests using the image passed.
